vllm/distributed/communication_op.py

```diff
@@ -13,17 +14,31 @@
                              get_tp_pynccl_communicator)
 
 
+@dataclass
+class GraphCaptureContext:
+    stream: torch.cuda.Stream
```

Comment:
How does this work for non-CUDA backends?

Reply:
For XPU, this will be `torch.xpu.Stream` .